content/docs/command-reference/add.md

```diff
@@ -374,33 +376,33 @@ outs:
     path: data.xml
 ```
 
+[link]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+[external cache]:
+  /doc/user-guide/managing-external-data#setting-up-an-external-cache
+
 ## Example: Transfer to remote storage
 
 When you have a large dataset in an external location, you may want to track it
-as if it was in your project, but without downloading it locally (for now). The
-`--to-remote` option lets you do so, while storing a copy
-[remotely](/doc/command-reference/remote) so it can be
-[pulled](/doc/command-reference/plots) later.
+in your <abbr>project</abbr> without downloading it (yet), for example if
+there's not enough space in your current environment.
```

Comment:
This example is almost the same as the one in the `import-url` ref. Since the latter has the added benefit of keeping the connection to the data source (for later `update`s) and to remove the docs maintenance surface, should we remove the example from this ref? I can still link to the example in `import-url` from the option description here.

Reply:
Makes sense to me!

Reply:
> > This example is almost the same as the one in the import-url ... should we remove the example from this ref?
>
> Makes sense to me!

UPDATE: After the latest round of updates I'm no longer 100% sure of this since for `add` you need to combine `--out` with `--to-remote`. So even when the situation is very similar, the actual recipe has that key difference. People who never use imports may prefer to stick to `add` and thus need to figure out that detail.

Keeping for now!

Reply:
Good point. Just to clarify, you can do `dvc add --to-remote` without `--out`:

```
$ dvc add --to-remote ../test.sh
100% Adding...|█████████████████████████████████████████████████|1/1 [00:00, 49.91file/s]
````

Reply:
> you can do dvc add --to-remote without --out

Hmm wait does that move the output to the workspace? ⌛

Reply:
UPDATE: Yeah it does... So `--to-remote` implies `--out .` ... 🤔

Reply:
> even when the situation is very similar, the actual recipe has that key difference (there's no -o in import-url)

I guess this wasn't that relevant after all since neither example uses `-o` now. The `add` example just notes that `-o .` is implied but that's not critical (already noted in the `--to-remote` option desc.)... We could remove it after all ⌛